### ISSUE TITLE
Mopub 5.13.1 java.lang.NullPointerException in com.mopub.mobileads.MoPubInline$1.onLoaded at line 112

### ISSUE THREAD

**Issue by glavatskikh** [mopub/mopub-android-sdk#426]
One line summary of the issue here

- [x] I am submitting a bug report for existing functionality
- [x] I visited https://developers.mopub.com/ and found no answer
- [x] I checked https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed
- [x] I checked to make sure that this issue has not already been filed

#### MoPub SDK Version:
5.13.1
#### Android Studio Version:
4.0.1
#### Gradle Version:
3.3.0
#### Device model and OS Version:
LML713DL
Pixel 3
REVVLPLUS C3701A
K88
LM-V405
SM-A505G
SM-J737T1
SM-G930VL
LM-X210(G)
moto g(7) supra
LG-LS777
SM-N970U
moto z4
LGL164VL
SM-G981U
moto e6 (XT2005DL)
9029W
Z982
Pixel XL
LM-X210CM
SM-G975U
SM-G930R6
SM-N960U
QTAXIA1
ONEPLUS A6013
5032W

7.0
7.1.2
7.1.1
6.0.1
8.1.0
9
8.0.0
10

#### Ad Unit IDs used in reproducing the issue:
All Ad Units with MoPubInline controller
#### Steps to reproduce the behavior:
Сall interstitial.destroy() when interstitial is loading process. 

#### Expected behavior:
SDK and app not crashed

#### Observed behavior:
SDK and app  crashed

#### Evidence:
**java.lang.NullPointerException
Attempt to invoke virtual method 'boolean android.os.Handler.postDelayed(java.lang.Runnable, long)' on a null object reference**

`ZygoteInit.java in com.android.internal.os.ZygoteInit.main at line 810`
`RuntimeInit.java in com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run at line 438`
`Method.java in java.lang.reflect.Method.invoke`
`ActivityThread.java in android.app.ActivityThread.main at line 6651`
`Looper.java in android.os.Looper.loop at line 164`
`Handler.java in android.os.Handler.dispatchMessage at line 106`
`chromium-Monochrome.aab-stable-418308170 in gB0.handleMessage at line 66`
`chromium-Monochrome.aab-stable-418308170 in Zo.b at line 2`
`MraidBridge.java in com.mopub.mraid.MraidBridge$4.onPageFinished at line 322`
`MraidBridge.java in com.mopub.mraid.MraidBridge.access$300 at line 53`
`MraidBridge.java in com.mopub.mraid.MraidBridge.handlePageFinished at line 406`
`MraidController.java in com.mopub.mraid.MraidController$4.onPageLoaded at line 190`
`MoPubInline.java in com.mopub.mobileads.MoPubInline$1.onLoaded at line 112`

**Comment by chauduyphanvu** [comment]
Thanks for letting us know! We have previously logged this issue internally. Will provide an update soon.

Vu Chau
MoPub Team

**Comment by chauduyphanvu** [comment]
@glavatskikh, we tried to reproduce this issue by calling `destroy()` while loading an interstitial but did not get this crash. Are you able to provide the code you are using that triggered this crash?

Thank you!

**Comment by glavatskikh** [comment]
@chauduyphanvu it's difficult to reproduce, but it happens when MoPubInline$1.**onLoaded** call the same time as MoPubInline.**onInvalidate**.
You can hardcode and check the stacktrace (is not good way):
``` java
mController.setMoPubWebViewListener(new BaseHtmlWebView.BaseWebViewListener() {
            @Override
            public void onLoaded(final View view) {
                // Honoring the server dimensions forces the WebView to be the size of the banner
                AdViewController.setShouldHonorServerDimensions(view);
                MoPubLog.log(LOAD_SUCCESS, ADAPTER_NAME);
                mLoadListener.onAdLoaded();
                onInvalidate(); // CALL DESTROY METHOD THIS
                mHandler.postDelayed(mAdExpiration, AD_EXPIRATION_DELAY);
            }
}
```

**Comment by chauduyphanvu** [comment]
Thanks! We've fixed this internally and will deploy the fix in the upcoming SDK version. Stay tuned!

**Comment by chauduyphanvu** [comment]
This has been fixed in [v5.14.0](https://github.com/mopub/mopub-android-sdk/commit/e8b8d865535855dacd65fdf741e1c0d5b7797ca5). Update and let us know if you continue seeing the issue!
